docs/docs/tutorials/using_dask.md

```diff
@@ -232,17 +287,7 @@ You can do this from your Jupyter notebook:
 cluster.close(shutdown=True)
 ```
 
-:::
-
-## Step 5 - Viewing the dashboard inside of JupyterLab
-
-The [Dask-labextension](https://github.com/dask/dask-labextension) provides a JupyterLab extension to manage Dask clusters,
-as well as embed Dask's dashboard plots directly into JupyterLab panes.
-Nebari includes this extension by default, elevating the overall developer experience.
-
-![Dask-labextension UI In JupyterLab - displays the demo notebook and a list of cluster settings](/img/tutorials/dask_labextension.png)
-
-## Step 6 - Using Dask safely
+## Using Dask safely
 
 If you're anything like us, we've forgotten to shut down our cluster a time or two. Wrapping the `dask-gateway` tasks in a
 context manager is a great practice that ensures the cluster is fully shutdown once the task is complete!
```

Comment:
I can't comment on that line, but below here on line 311 there is another mention of the dask env name that needs to be updated.

Reply:
Thanks, done!